app/src/pages/inside/common/defectTypeTooltip/defectTypeTooltip.scss

```diff
@@ -3,11 +3,11 @@
 .total-item {
   display: flex;
   justify-content: space-between;
-  box-sizing: border-box;
+  text-decoration: none;
```

Comment:
Which purpose of this swap?
Please see our [docs](https://github.com/reportportal/service-ui/blob/v5/app/docs/8-CSS.md)

Reply:
Fixed.